.evergreen/.evg.yml

```diff
@@ -1491,6 +1491,12 @@ tasks:
       tags: ["perf"]
       commands:
         - func: "bootstrap mongo-orchestration"
+          vars:
```

Comment:
Do you understand how this worked before adding these variables?

Reply:
Based on my understanding, we're using the same set of variables from matrix_spec:

{ auth: "noauth", ssl: "nossl", jdk: "jdk17", version: ["latest"], topology: "standalone", os: "linux" }

JDK 17 is the default in javaConfig.bash, and we've updated the database version. This should maintain the original workflow. Is there something I might be overlooking?"

Reply:
No, I get it now.  I didn't put it together that we need the variables after removing the matrix spec.